Content.Shared/_DV/CosmicCult/Components/CosmicGlyphComponent.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._DV.CosmicCult.Components;
+
+[RegisterComponent]
+public sealed partial class CosmicGlyphComponent : Component
+{
+    [DataField] public string GlyphName = "base";
```

Comment:
If `"base"` a valid name? Otherwise mark this field as required
```suggestion
    [DataField(required: true)] public string GlyphName = "base";
```